portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -7,30 +7,42 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
 
 @WebServlet("/login-status")
 public class LoginStatusServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html");
 
     UserService userService = UserServiceFactory.getUserService();
-    if (userService.isUserLoggedIn()) {
+    Boolean loggedIn = userService.isUserLoggedIn();
+
+    String json = "{";
```

Comment:
I'd recommend against building JSON documents piecewise from Strings: it seems both tedious and error-prone.

The JsonObject interface provides a fluent builder interface and a stronger guarantee that the final document is valid, well-formatted JSON.

https://docs.oracle.com/javame/8.0/api/json/api/com/oracle/json/JsonObject.html